src/utils/bindActionCreators.js

```diff
@@ -1,13 +1,18 @@
 import mapValues from '../utils/mapValues';
 
+function bindActionCreator(actionCreator, dispatch) {
+    return (...args) => dispatch(actionCreator(...args));
```

Comment:
Oh, sorry! Maybe a [`.editorconfig`](http://editorconfig.org/) file would be nice? This way most editors will recognize the proper indent / lineending style :)